docs/policy/balloons.md

```diff
@@ -76,6 +76,13 @@ Balloons policy parameters:
 - `ReservedPoolNamespaces` is a list of namespaces (wildcards allowed)
   that are assigned to the special reserved balloon, that is, will run
   on reserved CPUs. This always includes the `kube-system` namespace.
+- `AllocatorTopologyBalancing` affects selecting CPUs for new
+  balloons. If `true`, new balloons are created using CPUs on
+  package/die/NUMA with most free CPUs. This optimizes for performance
```

Comment:
just a nit, can we replace the order from `package/die/NUMA` to `NUMA/die/package`. Because, we go from bottom to top, while here it is the opposite.

Reply:
Yeah, I like it more in bottom-to-top order, too. Thanks, fixed!